plugins/parquet/README.md

```diff
@@ -77,5 +82,76 @@ To select the fields that you wish to partition your data on pass in a string of
 ```
 ParquetIO.HivePartition.Write(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
 
-ParquetIO.DirectoryPartition.Read(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
-```
\ No newline at end of file
+ParquetIO.DirectoryPartition.Write(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
+```
+
+## Apache Parquet/Arrow to ECL Type Mappings
+
+### ECL Record to Arrow Schema Mappings
+
+| ECL Record Type | Apache Arrow/Parquet Type |
+|-----------------|---------------------------|
+| BOOLEAN         | Boolean                   |
+| INTEGER         | Int64                     |
+| INTEGER8        | Int64                     |
```

Comment:
Since you show the mapping of INTEGER8 -> Int64 you can remove the default INTEGER -> Int64. It would be good to add a note that the size of the arrow type depends on number of bytes defined in the ECL record.

Reply:
Same comment for UNSIGNED.